cmd/algorand-indexer/main.go

```diff
@@ -58,6 +58,20 @@ var rootCmd = &cobra.Command{
 			err = pprof.StartCPUProfile(profFile)
 			maybeFail(err, "%s: start pprof, %v", cpuProfile, err)
 		}
+		if configFile != "" {
+			configs, err := os.Open(configFile)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "%v", err)
+				os.Exit(1)
+			}
```

Comment:
You can use the `maybeFail` helper for print + exit messages like this and the one below.